docs/source/asr/configs.rst

```diff
@@ -218,6 +218,31 @@ For example, a decoder config corresponding to a sub-word tokenization model sho
       vocabulary: []  # filled with vocabulary from tokenizer at runtime
 
 
+Dynamic Layer Freezing
+---------------------------------
+
+You can selectively freeze any modules inside a Nemo model by specifying a freezing schedule in the config. Freezing stops any gradient updates
```

Comment:
It just works for ASR models, not all nemo models.

Reply:
NA since we have now transferred to ModelPT